open-sauced/100-days-of-oss-template
# Add day 1

## Description


This PR modifies the journal to track Day 1 progress


## Related Tickets & Documents


## Mobile & Desktop Screenshots/Recordings


## Steps to QA
1. Open journal.md
2. Verify Day 1 progress has been added

## Tier (staff will fill in)


## What gif best describes this PR or how it makes you feel?
![excited gif](https://media0.giphy.com/media/v1.Y2lkPTc5MGI3NjExbm05enRiaG9ub2NwN3hnajZnem9kdW5tbHMwbnRjcHpjeTU5eDJmeCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/azaMjwRFm0vjNSd51t/giphy.webp)
